src/container/rpc.h

```diff
@@ -383,6 +386,17 @@ CRT_RPC_DECLARE(cont_prop_set, DAOS_ISEQ_CONT_PROP_SET, DAOS_OSEQ_CONT_PROP_SET)
 CRT_RPC_DECLARE(cont_acl_update, DAOS_ISEQ_CONT_ACL_UPDATE,
 		DAOS_OSEQ_CONT_ACL_UPDATE)
 
+#define DAOS_ISEQ_CONT_ACL_DELETE	/* input fields */	 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
False positive